src/components/Footer/Footer.js

```diff
@@ -1,14 +1,18 @@
-import { useContext } from 'react';
+import { useContext, useState, useEffect } from 'react';
 import ThemeContext from '../../context/themeContext';
 
-const Footer = (props) => {
+export default function Footer() {
   const theme = useContext(ThemeContext);
+  const [date , setDate] = useState();
+  const getYear = () =>  setDate(new Date().getFullYear())
+  useEffect(() => {
+      getYear();
+  }, [])

```

Comment:
This should be written as:
const date = new Date().getFullYear();`
without any state, effect - will be simpler.